de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -86,7 +89,7 @@ protected IStatus run(IProgressMonitor monitor) {
                 monitor
                     .beginTask("Starting session...", IProgressMonitor.UNKNOWN);
                 try {
-                    sessionManager.startSession(newResources);
+                    sessionManager.startSession(newResources, referencePointManager);
                     Set<JID> participantsToAdd = new HashSet<JID>(contacts);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq4IkEeD8QRkTSvD&open=AWcmJq4IkEeD8QRkTSvD)

[ISSUE_KEY]: AWcmJq4IkEeD8QRkTSvD